webhook_server/libs/github_api.py

```
@@ -291,9 +312,78 @@ async def get_pull_request(self, number: int | None = None) -> PullRequest | Non
+                username = reviewer.user.login
+            elif isinstance(reviewer, dict):
+                username = reviewer.get("login") or (reviewer.get("user") or {}).get("login")
+
+            if not username:
+                self.logger.warning(f"{self.log_prefix} Could not resolve username from reviewer: {reviewer}")
+                continue
+
+            try:
+                user_id = await self.unified_api.get_user_id(username)
+                reviewer_ids.append(user_id)
+            except Exception as ex:
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)